### ISSUE TITLE
Calc Parenthesis Bug with multiple properties

### ISSUE THREAD

**Issue by zackcote** [jakubpawlowicz/clean-css#511]
I'm having the same issue described in: https://github.com/jakubpawlowicz/clean-css/issues/496 but in the latest version of clean-css. 

Specifically though, it's only occurring in the presence of other properties. It's very odd. An example:

**Input**
```
.test {
    left: calc(99.99% * -1/3 - (30px - 30px * -1/3) + 30px);
}
.test2 {
    margin-right: 30px;
    left: calc(99.99% * 1/3 - (30px - 30px * 1/3) + 30px);
}
```

**Output (lines broken for clarity)**
```
.test{left:calc(99.99% * -1 / 3 - (30px - 30px * -1 / 3) + 30px)}
.test2{margin-right:30px;left:calc(99.99% * 1 / 3 - (30px - 30px * 1 / 3)+ 30px)}
```

Super-annoying, and i'm assuming it might have something to do with the (re)ordering of properties. I'll look into it myself, just wanted to share here in case anyone more familiar with the clean-css codebase feels like diving in first. Thanks!

**Comment by zackcote** [comment]
Also, disabling advanced options is a suitable fix in the meantime.

**Comment by jakubpawlowicz** [comment]
@zackcote That's puzzling. I'll take a look tomorrow morning.

**Comment by jakubpawlowicz** [comment]
@zackcote It's fixed in 3.1.9 - there's an ongoing issue (#429) I'm working on right now, so it'll be much better in upcoming 3.2.

**Comment by zackcote** [comment]
Much appreciated, thanks!
